NeoCoast/pis2019-chat-dashboard-frontend
# Disables buttons when input is disabled.

# :wrench: Bugfixes

## Disables send button

#### :pencil2: Description:

Only the input was disabled when we wanted to disable all input buttons.

#### How to test

- Finish a conversation and at the time of feedback, check you are unable to either send or input emojis or files using any of the buttons.
